skl2onnx/common/_container.py

```diff
@@ -34,8 +35,8 @@ def _get_operation_list():
                     found = g.groups()[0]
                     break
             if found is None:
-                raise RuntimeError("Unable to find an ONNX name in function "
-                                   "'{0}', source=\n{1}".format(k, source))
+                warnings.warn("Unable to find an ONNX name in function "
```

Comment:
It is needed?

Reply:
I talked to @wenbingl about this earlier -- a number of the operations in onnxconverter_common.onnx_ops are not being picked up by the regex command. Is there a better way to identify the operations without modifying the apply_op code for container.add_node('Name'...)? I think keras-onnx manually lists all the operators.

Reply:
(But yes, it is needed).